app/blueprints/assessments/routes.py

```diff
@@ -752,6 +752,16 @@ def activity_trail(application_id: str):
     scores = get_score_and_justification(
         application_id=application_id, score_history=True
     )
+    _ = [  # get sub_criteria name from id
+        score.update(
+            {
+                "sub_criteria_name": get_sub_criteria(
+                    application_id, score.get("sub_criteria_id")
+                ).name
+            }
+        )
+        for score in scores
+    ]
```

Comment:
Can we have a unit test to check the name is populated please?

Reply:
Added the test. Can you look again please?

Reply:
@srh-sloan